sources/3.0.0/sections/12-metadata.adoc

```diff
@@ -401,7 +401,7 @@ _False_ indicates the dataset *is* intended to be used for navigation.
 
 |Attribute
 |boundingBox
-|The extent of the datast limits
+|The extent of the dataset limits defined as a rectangle coincident with the outermost cell boundaries of the dataset. NoData cells included.
```

Comment:
I think there is no need to explicitly state that NoData/fill value cells must be included as, at least to me, the "outermost cell boundaries" implies this. I did however leave this in place in case there are differing opinions.

Reply:
SMA agree. It does not have to be stated that NoData cells included. It is enough with "outermost cell boundaries".

Reply:
Made a change accordingly.